x-pack/plugins/monitoring/public/plugin.ts

```diff
@@ -93,6 +93,10 @@ export class MonitoringPlugin
       category: DEFAULT_APP_CATEGORIES.management,
       mount: async (params: AppMountParameters) => {
         const [coreStart, pluginsStart] = await core.getStartServices();
+        const [, { AngularApp }] = await Promise.all([
+          pluginsStart.kibanaLegacy.loadAngularBootstrap(),
+          import('./angular'),
+        ]);
         const { AngularApp } = await import('./angular');
```

Comment:
@stratoula thanks for taking care of this.
Sorry, but I just realized that after applying the suggestion, the following line is not needed:
```suggestion
```

Reply:
oh damn I forgot it! Thanx